docs/guide/db-migrations.md

```diff
@@ -616,8 +619,8 @@ The migration command comes with a few command-line options that can be used to
 * `generatorTemplateFiles`: array (defaults to `[
         'create' => '@yii/views/createMigration.php',
         'drop' => '@yii/views/dropMigration.php',
-        'add' => '@yii/views/addMigration.php',
-        'remove' => '@yii/views/removeMigration.php',
+        'add_column' => '@yii/views/addMigration.php',
```

Comment:
done!

there is small issue in tests, in this lines https://github.com/pana1990/yii2/blob/9465-enhancements/tests/framework/console/controllers/MigrateControllerTestTrait.php#L155

and https://github.com/pana1990/yii2/blob/9465-enhancements/tests/framework/console/controllers/MigrateControllerTestTrait.php#L156 , sometimes it generates different timestamp, something idea to resolve this small issue?